py-polars/tests/unit/dataframe/test_df.py

```diff
@@ -1650,11 +1650,15 @@ def __repr__(self) -> str:
 def test_group_by_order_dispatch() -> None:
     df = pl.DataFrame({"x": list("bab"), "y": range(3)})
 
-    result = df.group_by("x", maintain_order=True).len()
-    expected = pl.DataFrame(
-        {"x": ["b", "a"], "len": [2, 1]}, schema_overrides={"len": pl.UInt32}
-    )
-    assert_frame_equal(result, expected)
+    for name in (None, "n", ""):
```

Comment:
I really want to avoid for-loops in tests if possible. If the test breaks, I don't know which test case actually ended up causing issues and it takes some effort to work that out.

Could you change this to a parametrize? Re-creating this dataframe a few times is a small price to pay - if it were a larger DF a fixture could be used.

Reply:
Done 👌